src/deltacode/__init__.py

```diff
@@ -48,16 +49,38 @@ class DeltaCode(object):
     the form of File objects) contained in those scans.
     """
     def __init__(self, new_path, old_path, options):
-        self.new = Scan(new_path)
-        self.old = Scan(old_path)
+        self.codebase1 = None
+        self.codebase2 = None
+        self.new_files_fingerprint = dict() # map of { {new_file1:fingerprint},{new_file2:fingerprint},...} it will be needed when we need the fingerprints
+        self.old_files_fingerprint = dict() # map of { {old_file1:fingerprint},{old_file2:fingerprint},...}
+        self.new_files_original_path = dict() #this keeps a map of the path of file with respect to original path 
```

Comment:
I think we can remove the fingerprints dicts here once your PR in common code is merged. 

Can you explain why `new_files_original_path` is necessary?

Reply:
yes.

Consider the `new_codebase` files as `new/a1.py` , `new/a2.py`  and the `old_codebase`structure as `old/a1.py` , `old/a4.py` . 
Now what `alignscan/fixtrees` does is it strips off the common directory path 
The  `new_codebase` would become `a1.py` `a2.py`    and similarly the `old_codebse` would become `a1.py` , `a4.py`

The original path would be lost, we are actually comapring the deltas based upon the new paths and then we are comparing the deltas based on their `sha1` values .
But in delta reports we need the `original paths ` and the `aligned path` so for that reason we are creating a dict of it .

Reply:
Lets just ignore/not do any alignment for this first implementation.

Reply:
Yeah